specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -846,7 +850,7 @@
         },
         "provisioningState": {
           "readOnly": true,
-          "$ref": "./network.json#/definitions/ProvisioningState",
+          "$ref": "#/definitions/NSPProvisioningState",
```

Comment:
What's the difference between these two provisioningState?

Reply:
NSPProvisioningState has two extra field: Accepting, Created.